src/Controls/src/Core/Items/ItemsView.cs

```diff
@@ -112,47 +110,15 @@ public int RemainingItemsThreshold
 		/// <include file="../../../docs/Microsoft.Maui.Controls/ItemsView.xml" path="//Member[@MemberName='AddLogicalChild']/Docs/*" />
 		public void AddLogicalChild(Element element)
 		{
-			if (element == null)
-			{
-				return;
-			}
-
-			_logicalChildren.Add(element);
-			element.Parent = this;
-			OnChildAdded(element);
-			VisualDiagnostics.OnChildAdded(this, element);
+			base.AddLogicalChildInternal(element);
```

Comment:
Why is this `base.`? Is there some duplicate overload in this class?

Reply:
Whoops, this was left over from a version where I was using method shadowing.